espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/MobileViewFlash.java

```diff
@@ -31,6 +31,7 @@ public void run() {
                 view.startAnimation(animation);
             }
         });
+        InstrumentationRegistry.getInstrumentation().waitForIdleSync();
```

Comment:
I'd rather assign the animation listener and set a semaphore/lock in `onAnimationEnd` handler. The current approach might not work as expected, because there might be other animations in the app